Vagrantfile

```diff
@@ -96,9 +108,9 @@ Vagrant.configure(2) do |config|
 
     winw1.vm.network :private_network, ip:"#{windows_node_ip}"
     winw1.vm.synced_folder ".", "/vagrant", disabled:true
-    winw1.vm.synced_folder "./sync/shared", "C:/sync/shared"
-    winw1.vm.synced_folder "./sync/windows/", "C:/sync/windows/"
-    winw1.vm.synced_folder "./forked", "C:/forked/"
+    winw1.vm.synced_folder "./sync/shared", "C:/sync/shared", type: "smb", smb_host: "10.20.30.20"
+    winw1.vm.synced_folder "./sync/windows/", "C:/sync/windows/", type: "smb", smb_host: "10.20.30.20"
+    winw1.vm.synced_folder "./forked", "C:/forked/", type: "smb", smb_host: "10.20.30.20"
```

Comment:
Is this host-agnostic? Does it work on Windows host with VirtualBox?

Reply:
This was a fix (temporary i believe) for the current `user mode` networking on Qemu as it has limitations with ping, ref [here](https://github.com/ppggff/vagrant-qemu#1-failed-to-create-shared-folder). There is still an issue with files not mounting correctly. Not sure if this is required when using virtualbox (with the networking mode it uses at the moment).